static/app/components/modals/debugFileCustomRepository/appStoreConnect/utils.tsx

```diff
@@ -116,3 +106,23 @@ export function getAppStoreErrorMessage(
     {}
   ) as Record<keyof StepOneData, string>;
 }
+
+export function getAppStoreValidationErrorMessage(
+  error: ValidationErrorDetailed
+): string {
+  switch (error.code) {
+    case 'app-connect-authentication-error':
+      return t(
+        'Credentials are invalid, missing, or expired. Check the entered App Store Connect credentials are correct and have not expired.'
```

Comment:
```suggestion
'Credentials are invalid, missing, or expired. Check that the credentials are correct and that they have not expired.'
```

Reply:
Is it possible for an app store's credentials to expire? 🧐

Reply:
good point: i was thinking it _could_ refer to credentials that have been redacted, but expired probably implies a temporal aspect to these that really isn't valid here. i'll remove the bits talking about expiration.